consensus/src/engine/impl_.rs

```diff
@@ -285,6 +291,11 @@ impl Engine {
                 continue;
             }
 
+            if is_paused {
+                is_paused = false;
+                self.committed_info_tx.send(MempoolOutput::Running).ok();
```

Comment:
Won't unwrapping here be better? Listener must be present; otherwise it can become a silent bug

Reply:
it's to shut down gracefully.
warning on channel close is logged in adapter with `scopeguard`, when consumer exits loop in spawned task